java/openjdk-jdk.xml

```diff
@@ -20,6 +20,9 @@ IcedTea project.</description>
     <command name="java">
       <runner interface="http://repo.roscidus.com/java/openjdk-jre"/>
     </command>
+    <command name="java-gui">
+      <runner interface="http://repo.roscidus.com/java/openjdk-jre" command="run-gui"/>
+    </command>
```

Comment:
A bit lower down we override "run" with:
```
      <command name="run">
        <runner interface="http://repo.roscidus.com/java/openjdk-jre"/>
        <arg>-cp</arg><arg>${JDK_CLASSPATH}</arg>
        <arg>com.sun.tools.javac.Main</arg>
      </command>
```
Do we need to override run-gui too in a similar way?